client/rest/http.go

```diff
@@ -29,13 +29,14 @@ import (
 	"strings"
 	"time"
 
+	"github.com/bloodhoundad/azurehound/v2/config"
 	"github.com/bloodhoundad/azurehound/v2/constants"
 )
 
 func NewHTTPClient(proxyUrl string) (*http.Client, error) {
 	transport := http.DefaultTransport.(*http.Transport).Clone()
-	transport.MaxConnsPerHost = 200
-	transport.MaxIdleConnsPerHost = 200
+	transport.MaxConnsPerHost = config.ColMaxConnsPerHost.Value().(int)
```

Comment:
nit: since this is configuration for a http client consider passing these values via function param to decouple from the app config singleton.

Reply:
I tried to make this change but it turned out to be a bigger lift than expected. The client creation here is nested under other constructors, and also different config structs than the main application, and it is going to require some significant plumbing or refactoring to get the desired result. @ddlees and I spoke about this and decided it's not worth it to make this change as part of this PR. It should definitely be re-visited in the future, though.